create TL3305 series tactile switch

andrewzuku opened the pull request:
I'd like to contribute a 3D model for the TL3305 series SMD tactile switch.

![screenshot](https://imgur.com/ghZrYXi.png "TL3305 series tactile switch").

See the matching pull request to the `kicad-packages3D` repo [here](https://github.com/KiCad/kicad-packages3D/pull/632) with far more details.

I've only supplied source for one of the three variants (differing only in actuator height). Let me know if you need source for each.

~ Andrew

andrewzuku committed:
create TL3305 series tactile switch

claassistantio commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/KiCad/kicad-packages3D-source?pullRequest=256) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/KiCad/kicad-packages3D-source?pullRequest=256) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/KiCad/kicad-packages3D-source?pullRequest=256) it.</sub>

andrewzuku committed:
Removed internal features and repositioned body

All bodies have been updated so that the leads end up touching the XY plane at z=0. Also, multiple bodies have been fused together.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/KiCad/kicad-packages3D-source?pullRequest=256) <br/>All committers have signed the CLA.

andrewzuku committed:
Rename STEP labels to match filenames

chschlue committed:
Remove deprecation warning

andrewzuku committed:
Merge branch 'master' into feature/TL3305-tactile-switch
